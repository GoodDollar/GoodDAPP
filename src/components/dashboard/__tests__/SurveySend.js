import React from 'react'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

import { getWebRouterComponentWithMocks } from './__util__'

describe('SurveySend', () => {
  it('renders without errors', () => {
    const SurveySend = getWebRouterComponentWithMocks('../SurveySend')
    const tree = renderer.create(<SurveySend />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const SurveySend = getWebRouterComponentWithMocks('../SurveySend')
    const component = renderer.create(<SurveySend />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
