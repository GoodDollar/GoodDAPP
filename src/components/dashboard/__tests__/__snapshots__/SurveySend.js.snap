// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurveySend matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "horizontal",
        "WebkitBoxPack": "center",
        "WebkitFlexDirection": "row",
        "WebkitJustifyContent": "center",
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "msFlexAlign": "center",
        "msFlexDirection": "row",
        "msFlexPack": "center",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "marginBottom": "8px",
          "marginTop": "8px",
          "paddingBottom": "8px",
          "paddingTop": "8px",
        }
      }
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(66,69,74,1.00)",
            "direction": "ltr",
            "fontFamily": "Roboto",
            "fontSize": "24px",
            "fontWeight": "700",
            "lineHeight": "30px",
            "textAlign": "center",
            "textDecoration": "none",
            "textTransform": "none",
          }
        }
      >
        Just a quick questionâ€¦
      </div>
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(66,69,74,1.00)",
            "direction": "ltr",
            "fontFamily": "Roboto",
            "fontSize": "16px",
            "fontWeight": "normal",
            "lineHeight": "22px",
            "textAlign": "center",
            "textDecoration": "none",
            "textTransform": "none",
          }
        }
      >
        (In order for us to improve)
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "horizontal",
        "WebkitBoxPack": "center",
        "WebkitFlexDirection": "row",
        "WebkitJustifyContent": "center",
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "msFlexAlign": "center",
        "msFlexDirection": "row",
        "msFlexPack": "center",
        "paddingTop": "8px",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(66,69,74,1.00)",
            "direction": "ltr",
            "fontFamily": "Roboto",
            "fontSize": "16px",
            "fontWeight": "700",
            "lineHeight": "22px",
            "textAlign": "center",
            "textDecoration": "underline",
            "textTransform": "none",
          }
        }
      >
        What are you paying for?
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignItems": "center",
        "WebkitBoxAlign": "center",
        "alignItems": "center",
        "msFlexAlign": "center",
        "paddingTop": "8px",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "width": "120px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "left",
            "WebkitFlexDirection": "row",
            "WebkitJustifyContent": "left",
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "left",
            "msFlexAlign": "center",
            "msFlexDirection": "row",
            "msFlexPack": "left",
          }
        }
      >
        <div
          aria-live="polite"
          className="css-cursor-18t94o4 css-view-1dbjc4n"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          role="button"
          style={
            Object {
              "borderBottomLeftRadius": "18px",
              "borderBottomRightRadius": "18px",
              "borderTopLeftRadius": "18px",
              "borderTopRightRadius": "18px",
              "cursor": "pointer",
              "msTouchAction": "manipulation",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "touchAction": "manipulation",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "borderBottomColor": "rgba(34,34,34,0.54)",
                "borderBottomLeftRadius": "10px",
                "borderBottomRightRadius": "10px",
                "borderBottomWidth": "2px",
                "borderLeftColor": "rgba(34,34,34,0.54)",
                "borderLeftWidth": "2px",
                "borderRightColor": "rgba(34,34,34,0.54)",
                "borderRightWidth": "2px",
                "borderTopColor": "rgba(34,34,34,0.54)",
                "borderTopLeftRadius": "10px",
                "borderTopRightRadius": "10px",
                "borderTopWidth": "2px",
                "height": "20px",
                "marginBottom": "8px",
                "marginLeft": "8px",
                "marginRight": "8px",
                "marginTop": "8px",
                "width": "20px",
              }
            }
          />
        </div>
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "normal",
              "lineHeight": "22px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          A product
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "left",
            "WebkitFlexDirection": "row",
            "WebkitJustifyContent": "left",
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "left",
            "msFlexAlign": "center",
            "msFlexDirection": "row",
            "msFlexPack": "left",
          }
        }
      >
        <div
          aria-live="polite"
          className="css-cursor-18t94o4 css-view-1dbjc4n"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          role="button"
          style={
            Object {
              "borderBottomLeftRadius": "18px",
              "borderBottomRightRadius": "18px",
              "borderTopLeftRadius": "18px",
              "borderTopRightRadius": "18px",
              "cursor": "pointer",
              "msTouchAction": "manipulation",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "touchAction": "manipulation",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "borderBottomColor": "rgba(34,34,34,0.54)",
                "borderBottomLeftRadius": "10px",
                "borderBottomRightRadius": "10px",
                "borderBottomWidth": "2px",
                "borderLeftColor": "rgba(34,34,34,0.54)",
                "borderLeftWidth": "2px",
                "borderRightColor": "rgba(34,34,34,0.54)",
                "borderRightWidth": "2px",
                "borderTopColor": "rgba(34,34,34,0.54)",
                "borderTopLeftRadius": "10px",
                "borderTopRightRadius": "10px",
                "borderTopWidth": "2px",
                "height": "20px",
                "marginBottom": "8px",
                "marginLeft": "8px",
                "marginRight": "8px",
                "marginTop": "8px",
                "width": "20px",
              }
            }
          />
        </div>
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "normal",
              "lineHeight": "22px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          A service
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "left",
            "WebkitFlexDirection": "row",
            "WebkitJustifyContent": "left",
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "left",
            "msFlexAlign": "center",
            "msFlexDirection": "row",
            "msFlexPack": "left",
          }
        }
      >
        <div
          aria-live="polite"
          className="css-cursor-18t94o4 css-view-1dbjc4n"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          role="button"
          style={
            Object {
              "borderBottomLeftRadius": "18px",
              "borderBottomRightRadius": "18px",
              "borderTopLeftRadius": "18px",
              "borderTopRightRadius": "18px",
              "cursor": "pointer",
              "msTouchAction": "manipulation",
              "overflowX": "hidden",
              "overflowY": "hidden",
              "position": "relative",
              "touchAction": "manipulation",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "borderBottomColor": "rgba(34,34,34,0.54)",
                "borderBottomLeftRadius": "10px",
                "borderBottomRightRadius": "10px",
                "borderBottomWidth": "2px",
                "borderLeftColor": "rgba(34,34,34,0.54)",
                "borderLeftWidth": "2px",
                "borderRightColor": "rgba(34,34,34,0.54)",
                "borderRightWidth": "2px",
                "borderTopColor": "rgba(34,34,34,0.54)",
                "borderTopLeftRadius": "10px",
                "borderTopRightRadius": "10px",
                "borderTopWidth": "2px",
                "height": "20px",
                "marginBottom": "8px",
                "marginLeft": "8px",
                "marginRight": "8px",
                "marginTop": "8px",
                "width": "20px",
              }
            }
          />
        </div>
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "normal",
              "lineHeight": "22px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          Other
        </div>
      </div>
    </div>
  </div>
</div>
`;
